ci-operator/step-registry/single-node/conf/e2e/graceful-shutdown/single-node-conf-e2e-graceful-shutdown-commands.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# set shutdown grace period toenable testing graceful shutdown
+yq e -i '.spec.kubeletConfig |= . + {"shutdownGracePeriod": "600s", "shutdownGracePeriodCriticalPods": "300s"}' "${SHARED_DIR}/manifest_single-node-reserve-sys-mem-kubeletconfig.yml"
```

Comment:
Can we rename `manifest_single-node-reserve-sys-mem-kubeletconfig.yml` in the e2e-conf step to a less specific name, now that it's used by multiple steps? Something like `manifest_single-node-kubeletconfig.yml`